examples/ml-pipelines-node-classification.ipynb

```diff
@@ -78,7 +78,7 @@
     "NEO4J_USERNAME = \"neo4j\"\n",
     "NEO4J_PASSWORD = \"test\"\n",
     "\n",
-    "gds = GraphDataScience(NEO4J_CONNECTION_URI, auth=(NEO4J_USERNAME, NEO4J_PASSWORD), database=\"neo4j\")\n",
```

Comment:
We still need this `database="neo4j"` part

Reply:
thanks, completely missed it...